tools/user/import/bareleasenotes/barelease.js

```diff
@@ -41,11 +41,11 @@ function getShortMonthName(monthNumber) {
 
 function printReleaseNotesHeader(currentMonth, currentYear) {
   return `---
-title: BigAnimal ${getMonthName(currentMonth)} ${currentYear} release notes
+title: Cloud Service ${getMonthName(currentMonth)} ${currentYear} release notes
 navTitle: ${getMonthName(currentMonth)} ${currentYear}
 ---
 
-BigAnimal's ${getMonthName(
+EDB Postgres AI Cloud Service's ${getMonthName(
```

Comment:
Should we add the trademark like in the other pages? 
```suggestion
EDB Postgres® AI Cloud Service's ${getMonthName(
```